index.css

```diff
@@ -0,0 +1,117 @@
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
Since this project is heavily dependent on flexbox it would be great to create custom classes so you you don't have to repeat yourself all the way